store.go

```diff
@@ -188,10 +187,7 @@ func (c *controller) watchNetworks() error {
 				lview := c.networks
 				c.Unlock()
 				for k, v := range lview {
-					global, _ := v.isGlobalScoped()
-					if global {
-						tmpview[k] = v
-					}
+					tmpview[k] = v
```

Comment:
This is added as part of delete processing to make sure we dont end up removing locally scoped networks incorrectly during a watch event.
With this change, we will lose that protection and the local networks will be removed when there is a watch event happens for the networks with global scope. Can you please confirm if you tried a scenario of both global and locally scoped objects ?